tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/bluetooth.py

```diff
@@ -9,11 +9,11 @@
 
 class Bluetooth(Base):
 
-    _bluetooth_checkbox_locator = (By.CSS_SELECTOR, '#bluetooth_v2 .bluetooth-status input')
-    _bluetooth_label_locator = (By.CSS_SELECTOR, '#bluetooth_v2 .bluetooth-status span')
+    _bluetooth_checkbox_locator = (By.CSS_SELECTOR, '#bluetooth_v2 .bluetooth-status gaia-switch')
```

Comment:
To prevent the duplication of `#bluetooth_v2`, we can make the `Bluetooth` app as a `PageRegion` instead.